packages/smooth_app/lib/pages/product/simple_input_text_field.dart

```diff
@@ -33,9 +33,7 @@ class SimpleInputTextField extends StatelessWidget {
           mainAxisSize: MainAxisSize.max,
           children: <Widget>[
             SizedBox(
-              width: constraints.maxWidth -
-                  LARGE_SPACE -
-                  (withClearButton ? MINIMUM_TOUCH_SIZE : 0),
+              width: 250,
```

Comment:
This will probably work on your device, but very likely not on every other. 

Could you try to fix the more dynamic variant instead of specifying a fixed value.

The constraints.maxWidth should be a good starting point

Reply:
Okay I will make it more dynamic

Reply:
We can use that minimum touch size of we are using it from edit product page ! We can use one more variable in simple_imput_field and we can use or check i guess ! I made it like that in my local device 😃